Source/Libraries/Common/Forms/LogConsoleForm.cs

```diff
@@ -41,13 +41,12 @@ public LogConsoleForm()
         /// <summary>
         /// Creates a LogConsoleForm using the global logger
         /// </summary>
-        /// <param name="maxLines">Maximum lines to display</param>
         /// <param name="layout">Layout</param>
         /// <param name="fileName">Optional file log</param>
-        public LogConsoleForm(int maxLines = 1000, Layout layout = null, string fileName = null)
```

Comment:
Can you revert this one and add an exception to the rule? While the core RTC doesn't make use of maxLines, plugins or implementations might. We don't have a stable plugin API yet (since so much is just public) and we want to work towards that, but as you've probably figured out by now, there's a lot of stuff we want to do. For now, we have the basic plugin system that we can use to make fun little tools without polluting the main RTC codebase, so some of this stuff is there to support that.

Reply:
Done. Do you also want me to revert my changes to `InitializeCustomLogger` and `LogConsole` and add exceptions there?

Reply:
Yes please

Reply:
Done